governance/roles.md

```diff
@@ -262,61 +276,111 @@ that role for some time informally, then proposed as an on-going role by Pull
 Request which must be approved by a majority of Chairs.  Process clarifications
 are welcome and PRs can be approved by any single Chair.
 
-This role may be subject to [lead rotations](#lead-rotations).
 
-### Security review facilitator
+### Security assessment facilitator
 
 [Security Reviews](/assessments) are part of the ongoing work of the group
-and led by a Security Assessment Facilitator, who will:
-
-* coordinate security review leads for upcoming security assessments.
-* identify and recommend security reviewers.
-* contribute to process improvements.
-* review and merge PRs in the /assessments directory (ensuring co-chair review
-  of significant process changes).
-* triage issues related to security assessments.
+and led by a security assessment facilitator (referred to in the rest of this
+sub-section as a facilitator). The facilitator is responsible for:
+
+* Ensuring that security reviews follow the assessment process.
+* Helping to bootstrap security assessments and reviews.
+* Determining assessment schedule of reviews, considering TOC requirements.
+
+The facilitator is responsible for coordinating with the TAG-Security co-chairs
+and deciding on the order in which security assessments (as submitted in the
+[GitHub issues tracker for
+TAG-Security](https://github.com/cncf/tag-security/issues?q=is%3Aopen+is%3Aissue+label%3Aassessment)
+will be addressed.  The facilitator is then responsible for reaching out to the
+project owners (via the GitHub issue said owners previously created in order to
+request an assessment), and coordinating the various requirements as outlined
+in the ["Joint Security Review" ticket
+template](https://github.com/cncf/tag-security/issues/new/choose). Once the
+maintainers/owners of the project have been identified, the facilitator is
+responsible for reaching out to the TAG-Security community at large (i.e.
+typically via attending the weekly general meetings or announcing to the
+community [via the `#tag-security`
+channel](https://cloud-native.slack.com/archives/CDJ7MLT8S)), and issue a
+call/request for reviewers (i.e. a single security assessment lead, and at
+least two additional security reviewers), and ensuring all reviewers [read the
+conflict of interest disclosure](../assessments/guide/security-reviewer.md) and
+sign-off on it in the GitHub ticket itself. 
```

Comment:
Right now, by rule, the chairs need to do an official sign off.  Are you saying that the facilitator should just make sure that the reviewers have stated their conflict statement or that the facilitator should actually sign off?

Reply:
I believe that the intent of the change is for the facilitator to do the signoff. In practice, I see how this proposed change streamlines getting started as is more representative of what happens: Facilitator ensures the reviewers have stated their conflict statement, either takes it at face value, or does quick research to corroborate the statements via LinkedIn and GitHub, and then chases the chairs to comment on the issue with signoff. A chair doing the signoff typically extends the lead time before starting. It has ceremonial value but it's impractical. The issue is public, if the minimum required number of statements is not met or there is no veracity in the statements, it's evident to everyone. The facilitator and the public can do the same check and arithmetic as the chair. The facilitator in its capacity has an interest in high-quality assessments so objectively are impartial.

Reply:
Okay, I'm fine with this.  

I do ask that the chairs sign off on this and merge it before I start taking on this myself (as the one in this role should have approval for the rule change). @sublimino @lumjjb @achetal01

Reply:
@PushkarJ in lieue of @lumjjb

Reply:
I do support this, i have no more juice now, but haven't found it to be that valuable since its publicly auditable as stated.